dll/win32/shell32/COpenWithMenu.cpp

```diff
@@ -1222,7 +1222,6 @@ VOID COpenWithMenu::AddApp(PVOID pApp)
     mii.fState = MFS_ENABLED;
     mii.wID = m_idCmdLast;
     mii.dwTypeData = const_cast<LPWSTR>(pwszName);
-    mii.cch = wcslen(mii.dwTypeData);
```

Comment:
Why? https://learn.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-menuiteminfow  seems to imply that setting cch is needed:
> MFT_STRING 0x00000000L | Displays the menu item using a text string. The  						dwTypeData member is the pointer to a null-terminated string, and the  						cch member is the length of the string. [...]

Reply:
You only need to set it when getting the string from a menu item, not when setting it.